libraries/app/application.cpp

```diff
@@ -463,6 +463,7 @@ namespace detail {
             wild_access.allowed_apis.push_back( "network_broadcast_api" );
             wild_access.allowed_apis.push_back( "history_api" );
             wild_access.allowed_apis.push_back( "crypto_api" );
+			wild_access.allowed_apis.push_back( "asset_api" );
```

Comment:
this was fixed in the last commit:
https://github.com/bitshares/bitshares-core/pull/220/commits/b50bba13c8dddbe7879b7b5888341bcc1b7a506e